application/cmdbabstract.class.inc.php

```diff
@@ -4735,11 +4735,15 @@ public static function DoBulkModify($oP, $sClass, $aSelectedObj, $sCustomOperati
 			$sCSSClass = $bResult ? HILIGHT_CLASS_NONE : HILIGHT_CLASS_CRITICAL;
 			$sChecked = $bResult ? 'checked' : '';
 			$sDisabled = $bResult ? '' : 'disabled';
+
+			$aErrorsToDisplay = array_map(function($sError) {
+				return utils::HtmlEntities($sError);
+			}, $aErrors);
 			$aRows[] = array(
 				'form::select' => "<input type=\"checkbox\" class=\"selectList\" $sChecked $sDisabled\"></input>",
 				'object' => $oObj->GetHyperlink(),
 				'status' => $sStatus,
-				'errors' => '<p>'.($bResult ? '' : implode('</p><p>', $aErrors)).'</p>',
+				'errors' => '<p>'.($bResult ? '' : utils::HtmlEntities(implode('</p><p>', $aErrorsToDisplay))).'</p>',
```

Comment:
Now that you entities encoded `$aErrors` on line 4739, you need to remove encoding on this line, otherwise it will be double encoded and the problem will remain.

```suggestion
				'errors' => '<p>'.($bResult ? '' : implode('</p><p>', $aErrorsToDisplay)).'</p>',
```